source/_integrations/brunt.markdown

```diff
@@ -10,36 +10,14 @@ ha_codeowners:
 ha_domain: brunt
 ha_platforms:
   - cover
+ha_config_flow: true
 ---
 
 The `brunt` platform allows one to control Blind Engines by [Brunt](https://www.brunt.co). To use this sensor, you need a Brunt App Account. All Brunt Blind devices registered to your account are automatically added to your Home Assistant with the names given them through the Brunt app.
 
 ## Configuration
```

Comment:
Check the preview build linked in the CI checks. This header is a duplicate. It's already included by the include below.